ui/pages/routes/routes.component.js

```diff
@@ -122,6 +96,66 @@ import {
   showOnboardingHeader,
 } from './utils';
 
+// Begin Lazy Routes
+const OnboardingFlow = React.lazy(() =>
+  import('../onboarding-flow/onboarding-flow'),
+);
+const Lock = React.lazy(() => import('../lock'));
```

Comment:
Minor, but something I did in my POC was create a helper function for this so we had an easy injection point for logging and timing.

https://github.com/MetaMask/metamask-extension/compare/develop...poc/react-lazy-load#diff-2466ed85b744b657eb0b324376eef5486080bf2b897eaa3af9b8c3ed0184ebeaR117

Reply:
This is a fascinating thought, and maybe the helper function could even handle the non-default-export thing.